spec/integration/lib/auth0/auth0_client_spec.rb

```diff
@@ -62,13 +62,12 @@
   context 'client headers' do
     let(:client) { Auth0::Client.new(v2_credentials.merge(access_token: 'abc123', domain: 'myhost.auth0.com')) }
     let(:headers) { client.headers }
-
-    let(:base64_token) do
-      Base64.urlsafe_encode64('{"name":"ruby-auth0","version":"' + Auth0::VERSION + '"}')
+    let(:telemetry) do
```

Comment:
Can you update this to use `{}` syntax, as this repo seems to use more of a Weirich-style block syntax? (meaning it will also match the above)

Reply:
@machuga - Fixed up!